financialaid/serializers.py

```diff
@@ -0,0 +1,54 @@
+"""
+Serializers from financialaid
+"""
+import datetime
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from rest_framework.fields import IntegerField, FloatField, CharField
+
+from courses.models import Program
+from dashboard.models import ProgramEnrollment
+from financialaid.backend import determine_tier_program, determine_auto_approval
+from financialaid.models import FinancialAid
+
+
+class FinancialAidSerializer(serializers.Serializer):
+    """
+    Serializer for Financial Aid objects
+    """
+    original_income = FloatField(min_value=0)
+    original_currency = CharField()
+    program_id = IntegerField()
+
+    def validate(self, data):
+        """
+        Validators for this serializer
+        """
+        data["program"] = get_object_or_404(Program, pk=data["program_id"])
```

Comment:
also I would prefer to raise custom exceptions so we can intercept them in the view and return proper meaningful errors to the UI